pyproject.toml

```diff
@@ -7,7 +7,112 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools_scm]
+[tool.setuptools_scm] # this section must exist
+
+[project]
+name = "compliance-trestle"
+description = "Tools to manage & autogenerate python objects representing the OSCAL layers/models"
+authors = [
+    {name = "OSCAL Compass", email = "oscal-compass-oversight@googlegroups.com"}
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Information Technology",
+    "Operating System :: POSIX",
+    "Operating System :: Microsoft",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+readme = "README.md"
+license = "Apache-2.0"
+license-files = ["LICENSE"]
+dynamic = ["version"]
+requires-python = ">=3.9"
+dependencies = [
+    "attrs",
+    "ilcli",
+    "cryptography==44.0.2",
+    "paramiko==3.5.0",
+    "ruamel.yaml",
+    "furl",
+    "pydantic[email]>=2.0.0",
+    "python-dotenv>=0.10.4",
+    "datamodel-code-generator[http] == 0.25.3",
+    "python-frontmatter",
+    "pywin32 >= 1.0;platform_system=='Windows'",
+    "defusedxml",
+    "openpyxl~=3.0",
+    "Jinja2 == 3.1.6",
+    "cmarkgfm==2024.1.*", ## Update regularly
+    "orjson",
+    "requests>=2.32.2",
+    "importlib_resources"
+]
+
+[project.optional-dependencies]
+dev = [
+    "pytest>=5.4.3",
+    "pytest-cov>=2.10.0",
+    "pytest-xdist",
+    "pre-commit>=2.4.0",
+    "setuptools>=61",
+    "urllib3==1.26.19",
+    "wheel",
+    "yapf",
+    "python-semantic-release>=9.8.0",
+    "pep8-naming",
+    "pytest-random-order",
+    "python-dateutil",
+    "mypy",
+    "types-PyYAML",
+    "types-paramiko",
+    "types-requests",
+    "types-setuptools",
+    ## Docs website
+    "mike",
+    "mkdocs>=1.6.0",
+    "mkdocs-awesome-pages-plugin",
+    "mkdocstrings[python]>=0.25.2",
+    "mkdocs-htmlproofer-plugin",
+    "mkdocs-material",
+    "markdown-include",
+    "mkdocs-minify-plugin",
+    "mkdocs-git-revision-date-localized-plugin",
+    "pymdown-extensions",
+    "livereload",
+    "pillow",
+    "cairosvg",
+    ## Constrain system
+    "pylint",
+    # Checking repo after docs generation.
+    "gitpython"
+]
+
+[project.urls]
+Homepage = "https://oscal-compass.github.io/compliance-trestle"
+
+[project.scripts]
+trestle = "trestle.cli:run"
+
+[tool.distutils.bdist_wheel]
```

Comment:
I noticed there was a message in CI that `tool.distutils`  in `pyproject.toml` is experimental.